help-docs/marketplace/apps/ChainAegis.md

```diff
@@ -0,0 +1,33 @@
+**About this app:**
+
+ChainAegis provide Rugpull alert, security incident alert, and you can check an address's reputation and risk types. Of course, you can also apply for other security services such as smart contract auditing, on-chain analysis, asset tracking, and data APIs.
+
+**To get started:**
+
+1. Install the ChainAegis Miniapp from the Collab.Land Marketplace
+
+2. Your community members will get the latest 5 Rugpull and 5 security incidents through _/chainaegis alert_.
+
+3. Your community members can check whether the address is risk through _/chainaegis check_, and they can also report addresses and risk details through _/chainaegis report_.
+
+4. Your community members can apply for other security services such as smart contract auditing, on-chain analysis, asset tracking, data apis through _/chainaegis request_
+
+**Commands**
+
+| **Command** | **Description** |
+| --- | --- |
+| /chainaegis alert | Get the latest 5 Rugpull and 5 security incidents. |
```

Comment:
wrap in \` please to make explicit for each command

eg. /chainaegis alert --> \`/chainaegis alert\` renders as `/chainaegis alert`

Reply:
I'll warp all commands in `